src/plugins/displayLayout/CustomStringFormatter.js

```diff
@@ -0,0 +1,33 @@
+import printj from 'printj';
+
+export default class CustomStringFormatter {
+    constructor(openmct, itemFormat = null) {
```

Comment:
Just curious if there is a reason for using `null` instead of `undefined`? They have subtly different meanings, and to avoid confusion in this project we (rightly or wrongly) tend to just use `undefined`. 

What we probably *should* be doing is using `undefined` for unassigned variables, and `null` for "nothing" assignments, but we would have to be consistent, and it would require a pretty significant refactoring of the codebase. Right now we are explicitly testing for `undefined` in a lot of places.

Reply:
Yes agree with, 
> What we probably _should_ be doing is using `undefined` for unassigned variables, and `null` for "nothing" assignments.

In an instance of a class it's easier to compare null ("nothing") values rather than `undefined || null`, like forgot to pass values while instantiating?.

Reply:
@nikhilmandlik Agree that the distinction is occassionally important, but we don't use it in this codebase. I think we should remain consistent and just use `undefined`. Developers are not expecting to check for `null` in our codebase and `undefined !== null`.

Reply:
removed default `null`.